reanahub/reana-commons
# api_client: strip whitespaces from the image specification

addresses https://github.com/reanahub/reana-job-controller/issues/364

To test:
- Follow instructions described [here](https://github.com/reanahub/reana-client/pull/604)
- Run the workflow, it should finish as expected
